config/application.rb

```diff
@@ -39,6 +39,21 @@ class Application < Rails::Application
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
 
+    # RAILS 7 CONFIG START
+    # DEPRECATION WARNING: ActiveSupport::TimeWithZone.name has been deprecated and
+    # from Rails 7.1 will use the default Ruby implementation.
+    config.active_support.remove_deprecated_time_with_zone_name = false
```

Comment:
So before we can upgrade to 7.1, we'll need to flip this to true and fix the tests?

Reply:
Looks that way... @stevenjcumming  chased this down and can probably speak to it better